test/integration/suite_test.go

```diff
@@ -309,3 +313,47 @@ func waitForBpfdConfigDelete(ctx context.Context, env environments.Environment)
 		}
 	}
 }
+
+func waitForDataplaneReadiness(ctx context.Context, env environments.Environment) error {
+	for {
+		time.Sleep(2 * time.Second)
+		select {
+		case <-ctx.Done():
+			if err := ctx.Err(); err != nil {
+				return fmt.Errorf("context completed while waiting for dataplane ready: %w", err)
+			}
+			return fmt.Errorf("context completed while waiting for dataplane ready")
+		default:
+			fmt.Println("INFO: waiting for dataplane ready")
+
```

Comment:
Right before we call this function we already print an info line that essentially matches this, so as written we would end up with:

```console
INFO: waiting for Dataplane readiness
INFO: waiting for dataplane ready
```

and then depending on how long it takes, maybe:

```
INFO: waiting for Dataplane readiness
INFO: waiting for dataplane ready
INFO: waiting for dataplane ready
INFO: waiting for dataplane ready
```

I suggest we remove this line here, and instead emit additional and specific info as necessary further down:

```suggestion
```